app-laptop/i8kutils/files/i8kmon.init

```diff
@@ -0,0 +1,19 @@
+#!/sbin/openrc-run
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+depend() {
+	after modules
+}
+
+command="/usr/bin/i8kmon"
+command_args="--nouserconfig"
+command_background=true
+pidfile="/run/${RC_SVCNAME}.pid"
+
+start_pre() {
+	if [ ! -e /proc/i8k ]; then
```

Comment:
You could use `[[ ]]`, otherwise LGTM!

Reply:
I thought we needed OpenRC scripts to be POSIX-compliant, but maybe it's different with openrc-init or something? (I've never really poked at it that much)

Reply:
Yeah no [[ ]] in there, openrc claims compatibility with busybox sh, ash, dash, bash, mksh, and zsh, and then our scripts break that.

Reply:
Well damn, someone should tell lxd's maintainer that so people don't learn bad examples from there :P 
https://gitweb.gentoo.org/repo/gentoo.git/tree/app-emulation/lxd/files/lxd-4.0.0.initd#n31

Reply:
We have a tracker too, no that it seen much activity [bug #609070](https://bugs.gentoo.org/609070)